ansible/configure-rhel-7.yml

```diff
@@ -4,3 +4,5 @@
     ansible_python_interpreter: /usr/bin/python2
   roles:
     - configure_rhel_7
+  when:
+    - ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
```

Comment:
This syntax is broken. `when` directives can't be applied to entire plays.

Reply:
Fixed